### ISSUE TITLE
Error when using Xrandr in monitor-mode

### ISSUE THREAD

**Issue by TijZwa** [Xpra-org/xpra#3743]
I'm using Xpra 5.x (Beta) in monitor-mode.
This works fine, but when I resize the screen using Xrandr (e.g. `DISPLAY=:10 xrandr --output DUMMY0 --mode 800x600`) the event below is logged.

The screen resizes fine.
```
2023-01-19 13:37:59,349 XError: <class 'KeyError'>, 'geometry'
Traceback (most recent call last):
  File "/usr/lib64/python3.11/site-packages/xpra/x11/desktop/monitor_server.py", line 151, in do_reconfigure
    self.reconfigure_monitors()
  File "/usr/lib64/python3.11/site-packages/xpra/x11/desktop/monitor_server.py", line 214, in reconfigure_monitors
    self.apply_monitor_config(defs)
  File "/usr/lib64/python3.11/site-packages/xpra/x11/desktop/monitor_server.py", line 266, in apply_monitor_config
    RandR.set_crtc_config(monitor_defs)
  File "xpra/x11/bindings/randr_bindings.pyx", line 1001, in xpra.x11.bindings.randr_bindings.RandRBindingsInstance.set_crtc_config
KeyError: 'geometry'
```

**Comment by TijZwa** [comment]
Yes, this happens with my own client but also the latest Xpra beta's.

**Comment by totaam** [comment]
Should be fixed in dab86890ef773fc45dd82809a50d83564753a2b3 - @TijZwa can you confirm?

**Comment by totaam** [comment]
I assume that this is fixed.
